nixos/modules/services/misc/taskchampion-sync-server.nix

```diff
@@ -45,6 +45,11 @@ in
         default = 14;
       };
     };
+    allow-client-ids = lib.mkOption {
```

Comment:
Usually, nixos module options are named using camelCase.  There are exceptions like RFC42, but I think we should use camelCase here.

Reply:
Done.